common/event_modifiers.txt

```diff
@@ -1581,6 +1581,12 @@ no_infamy = {
 	
 	icon = 46
 }
+unciv_nation_5 = {
+	icon = 16
+}
+unciv_nation_7 = {
+	icon = 16
+}
```

Comment:
Use country flags, not modifiers

Reply:
I used both, because modifiers get shown in the politics screen, but checking for them in the pop file would crash the game, but country flags don't get shown or do they?

Reply:
Country flags get shown in conditions. So no need for the extra modifier. Just run with the country flag instead